spinn_front_end_common/utilities/constants.py

```diff
@@ -19,6 +19,7 @@
 # conversion from words to bytes
 BYTES_PER_WORD = 4
 BYTES_PER_3_WORDS = 12
```

Comment:
I think the only instance of BYTES_PER_3_WORDS was replaced with BYTES_PER_4_WORDS, so perhaps this could be removed?

Reply:
Checked and removed